public/index.html

```diff
@@ -45,6 +46,29 @@
                     $scope.ideas = response.data;
                 });
     });
+    app.controller('voteCtrl', function($scope, $window, $http) {
+    $scope.myFunction = function() {
+        $http({
+          method: 'POST',
+          url: '/vote',
+          headers: {
+            'Content-Type': "application/json"
+          },
+          // TODO remove hard coded urls
+          data: { "person" : "http://localhost:8080/people/1",
```

Comment:
You can omit domain part and use absolute URLs starting with root: `/people/1` instead of `http://localhost:8080/people/1`